src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/impl/GraphQLEndpointInitializerImpl.java

```diff
@@ -0,0 +1,232 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.impl;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.apache.jena.atlas.json.JsonObject;
+import org.apache.jena.atlas.json.JsonValue;
+
+import se.liu.ida.hefquin.engine.federation.GraphQLEndpoint;
+import se.liu.ida.hefquin.engine.federation.access.FederationAccessException;
+import se.liu.ida.hefquin.engine.federation.access.GraphQLInterface;
+import se.liu.ida.hefquin.engine.federation.access.GraphQLRequest;
+import se.liu.ida.hefquin.engine.federation.access.JSONResponse;
+import se.liu.ida.hefquin.engine.federation.access.impl.iface.GraphQLInterfaceImpl;
+import se.liu.ida.hefquin.engine.federation.access.impl.req.GraphQLRequestImpl;
+import se.liu.ida.hefquin.engine.federation.access.impl.reqproc.GraphQLRequestProcessor;
+import se.liu.ida.hefquin.engine.federation.access.impl.reqproc.GraphQLRequestProcessorImpl;
+import se.liu.ida.hefquin.engine.utils.Pair;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.GraphQLEndpointInitializer;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.GraphQLEntrypoint;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.GraphQLField;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLEntrypointImpl;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLEntrypointType;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldImpl;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldType;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.query.GraphQLQuery;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.query.impl.GraphQLQueryImpl;
+
+public class GraphQLEndpointInitializerImpl implements GraphQLEndpointInitializer {
+
+    @Override
+    public GraphQLEndpoint initializeEndpoint(final String url) throws FederationAccessException {
+        final GraphQLInterface iface = new GraphQLInterfaceImpl(url);
+        final GraphQLEndpoint tmpEndpoint = new GraphQLEndpointImpl(null, null, iface);
+        final GraphQLRequestProcessor requestProcessor = new GraphQLRequestProcessorImpl(5000,5000);
```

Comment:
Maybe there should be a way to customize these parameters, instead of hardcoding 5000s